src/IconButton/_IconButton.scss

```diff
@@ -15,27 +15,49 @@ $btn-icon-diameter-inline: calc(#{$line-height-base}em + 0.1em) !default;
 
 $btn-icon-bg: transparent !default;
 $btn-icon-accent-color: white !default;
+$btn-active-bg: #454e52 !default;
 
 @mixin btn-icon-size($icon-size) {
     width: $icon-size;
     height: $icon-size;
 }
 
-@mixin btn-icon-variant($icon-color, $accent-color) {
-    background-color: $btn-icon-bg;
-    color: $icon-color;
-
-    &.hover,
-    &:hover {
+@mixin btn-icon-variant($icon-color, $accent-color, $active-state: false) {
+    @if $active-state {
         background-color: $icon-color;
         color: $accent-color;
-    }
 
-    &.focus,
-    &:focus {
+        &.hover,
+        &:hover,
+        &.focus,
+        &:focus,
+        &.active,
+        &:active {
+            background-color: $icon-color;
+            color: $accent-color;
+        }
+    } @else {
         background-color: $btn-icon-bg;
         color: $icon-color;
-        box-shadow: inset 0 0 0 $btn-focus-width $icon-color;
+
+        &.hover,
+        &:hover {
+            background-color: $icon-color;
```

Comment:
nit: The `background-color` and `color` properties between these `.hover/:hover` styles and the below `.active/:active` classes are the same. Should these be consolidated, e.g.:

```scss
&.hover,
&:hover,
&.active,
&:active {
  background-color: $icon-color;
  color: $accent-color;
}
```

Reply:
ah ok only focus needs the conditional so will refactor